website/docs/d/servicebus_subscription.html.markdown

```diff
@@ -57,6 +57,14 @@ output "servicebus_subscription" {
 
 * `forward_dead_lettered_messages_to` - The name of a Queue or Topic to automatically forward Dead Letter messages to.
 
+* `client_scoped_subscription_enabled` - The subscription has an affinity to the client id or not. 
```

Comment:
the name here uses scoped, but the description talked about it being having an affinity? should this be
```suggestion
* `client_scoped_subscription_enabled` - The subscription has an affinity to the client id or not. 
```
```suggestion
* `client_affinity_enabled` - Does the subscription has an affinity to a client id?
```

Reply:
I used the word by reading the article: https://docs.microsoft.com/en-us/azure/service-bus-messaging/java-message-service-20-entities#additional-concepts-for-java-message-service-jms-20-subscriptions. The article talks about whether the subscription is scoped to a client or not.

Reply:
thanks, i agree after reading those docs it does sound like scoped is better